lib/src/Uncertainty/Distribution/MarshallOlkinCopula.cxx

```diff
@@ -285,6 +287,12 @@ Point MarshallOlkinCopula::getRealization() const
   return realization;
 }
 
+Bool MarshallOlkinCopula::isContinuous() const
+{
+  if (alpha_ == beta_) return true;
+  return false;
```

Comment:
return alpha_ == beta_

Reply:
Actually I was wrong, the Marshall-Olkin copula is never continuous (except of course in the case where alpha = beta = 0 which is the independent copula).